region/brinstar/red/Beta Power Bomb Room.json

```diff
@@ -272,10 +272,24 @@
       "link": [1, 3],
       "name": "Morph Power Beam Sidehopper Kill",
       "requires": [
-        "Morph"
+        "Morph",
+        {"or": [
+          "canTrickyDodgeEnemies",
+          {"and": [
+            "canDodgeWhileShooting",
+            {"enemyDamage": {
+              "enemy": "Sidehopper",
+              "type": "contact",
+              "hits": 2
+            }},
+          ]}
+        ]}
```

Comment:
can also consider `canCameraManip` to get the enemies one at a time

Reply:
Agreed.